includes/admin/functions.php

```diff
@@ -96,7 +96,7 @@ function amp_admin_get_preview_permalink() {
  */
 function amp_add_customizer_link() {
 	/** This filter is documented in includes/settings/class-amp-customizer-design-settings.php */
-	if ( ! apply_filters( 'amp_customizer_is_enabled', true ) || current_theme_supports( AMP_Theme_Support::SLUG ) ) {
+	if ( ! apply_filters( 'amp_customizer_is_enabled', true ) || amp_is_canonical() ) {
```

Comment:
I think this change can be reverted because there should not be an AMP panel in the Customizer if the theme has AMP support. In other words, `amp` theme support means that it is Standard or Transitional mode. Only in classic Reader mode is there an AMP panel.